Source/Shared/arcana/functional/inplace_function.h

```diff
@@ -351,35 +371,43 @@ namespace stdext
             return (*functor)(std::forward<ArgsT>(args)...);
         }
 
-        template<typename FunctorT>
-        static void manage(void* dataPtr, const void* fromPtr, Operation op)
+        template<typename FunctorT, bool SupportsCopying>
+        struct manage_function
         {
-            FunctorT* thisFunctor = reinterpret_cast<FunctorT*>(dataPtr);
-            switch (op)
+            static void call(void* dataPtr, const void* fromPtr, Operation op)
             {
-            case Operation::Destroy:
-                thisFunctor->~FunctorT();
-                break;
-            case Operation::Copy:
+                FunctorT* thisFunctor = reinterpret_cast<FunctorT*>(dataPtr);
+                switch (op)
+                {
+                case Operation::Destroy:
+                    thisFunctor->~FunctorT();
+                    break;
+                case Operation::Move:
+                {
+                    FunctorT* source = (FunctorT*)fromPtr;
+                    new (thisFunctor) FunctorT(std::move(*source));
+                    break;
+                }
```

Comment:
nit: I would indent this and use {} on both cases.

Reply:
Can do. However, I didn't really change this code, just moved it somewhere, and this was the style of the original source file. Given that, is it still your preference that we change the style?

Reply:
yes, it looks weird.